### ISSUE TITLE
Request: "Would you like to save changes?"

### ISSUE THREAD

**Issue by jschwalbe** [timotto/ardumower-modem#8]
It took me a few tries before I figured out that the little blue [ √ ] checkbox was required before uploading firmware. I couldn't figure out why my settings weren't saving. Maybe I'm just a dope (yeah, probably), maybe it should prompt "Do you want to save changes before leaving this page?", or just *maybe* it's Maybelline.

**Comment by timotto** [comment]
I agree it's bad UX. I will use this as a feature request to make saving pending changes more obvious.
